### ISSUE TITLE
Report if tv is in pause or play mode

### ISSUE THREAD

**Issue by GitMastic** [ollo69/ha-samsungtv-smart#350]
Detect if the mode is "pause" in tv but especially on media/apps. Since I sometimes pause, do something else and then I forget about it, it would be convient to make a script to turn it off after some time or issue an alarm

**Comment by bvdf0251** [comment]
Seconded,  this would be a great thing to have.

**Comment by FrankTub** [comment]
Yeah I already thought so but was worth a shot. My use case would be if the tv is paused then the lights turn on again (with some more conditions). And when the TV starts playing again specific lights turn off again.

Also tried some other approaches like network sniffing but could not get it to work with my setup. Perhaps someone else has a suggestion how to achieve this?
